### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by wang-503** [share121/inter-knot#2772] [其他]
1.怎么在手机上加速GitHub啊，我看watt toolkit妹有手机版呐求大佬们助我
      ->求watt toolkit手机版
2.在电脑版绳网上设置封面我都一知半解什么url啥的我是真不懂啊，手机版就更宕机了
      ->求如何在手机版绳网上设置封面

**Comment by CeceliaHarex** [comment]
这个委托我接了！
[watt toolkit官方蓝奏云下载，密码1234](https://wwe.lanzouy.com/b01v4izfa)
设置封面我给你放了视频，如图：

https://github.com/user-attachments/assets/14ce9a01-1b46-4cfd-965a-4eb42032b021

**Reply by CeceliaHarex** [reply]
快给我发3000[菲林](https://baike.baidu.com/item/%E8%8F%B2%E6%9E%97/64622252)

**Reply by wxgl** [reply]
good👍🏻

**Comment by CeceliaHarex** [comment]
对了，忘了跟你说了，那个watt toolkit是安卓版的

**Comment by zhixuan45** [comment]
直接用工具就行

**Comment by wang-503** [comment]
感谢所有大佬谢谢（感动
